bin/tx-prover/src/commands/proxy.rs

```diff
@@ -0,0 +1,55 @@
+use clap::Parser;
+use pingora::{
+    apps::HttpServerOptions, lb::LoadBalancer as PingoraLoadBalancer, prelude::Opt, server::Server,
+};
+use pingora_proxy::http_proxy_service;
+
+use crate::{
+    proxy::LoadBalancer,
+    utils::{load_config_from_file, setup_tracing},
+};
+
+/// Starts the proxy defined in the config file.
+#[derive(Debug, Parser)]
+pub struct StartProxy;
+
+impl StartProxy {
+    /// Starts the proxy defined in the config file.
+    ///
+    /// This method will first read the config file to get the list of workers to start. It will
+    /// then start a proxy with each worker as a backend.
+    pub fn execute(&self) -> Result<(), String> {
+        setup_tracing();
```

Comment:
very minor nit (feel free to disregard): This looks like it could be performed on program startup